src/preload.js

```diff
@@ -6,7 +6,6 @@ const { getLogger } = require('./logger');
 
 function preload() {
   window.Workbench = {
-    isDevMode: !!process.defaultApp,
```

Comment:
This never actually worked. Since this file runs in a renderer process it never has `process.defaultApp` defined. Also at this point nothing in the renderer needs to be aware of dev vs production modes.